src/backend/base/langflow/components/models/OllamaModel.py

```diff
@@ -55,8 +55,9 @@ def update_build_config(self, build_config: dict, field_value: Any, field_name:
 
         return build_config
 
-    def get_model(self, url: str) -> list[str]:
+    def get_model(self, base_url_value: str) -> list[str]:
```

Comment:
nit: could just call this base_url

Reply:
I changed it such that the endpoint url of the models are formed inside the function.
Earlier the input to the function was the url to the tags/models, which is ok, but in this way it would be more clear that the model is loaded from the base url and the endpoint url is framed without error inside the get_model function. Should I revert back to the previous method and have the model url endpoint before calling the function ?

Reply:
Nah, that's fine